ga4gh/frontend.py

```diff
@@ -9,15 +9,19 @@
 
 import os
 import traceback
+import datetime
 
 import flask
 import flask.ext.api as api
```

Comment:
Sorry, my bad. I don't think it's worth mandating flake8 checks, as they don't like some of the fancier things we're doing with _protocoldefinitions.py.